.github/ISSUE_TEMPLATE/support.yml

```diff
@@ -0,0 +1,35 @@
+# Documentation: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms
+# Syntax: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
+
+name: Support Request
+description: File a support request.
+title: "<title>"
+labels: ["support-request"]
+type: Task
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
```

Comment:
This is not a bug report, maybe we should take the opportunity to add details about the expectations / process